test_vm/tests/move_partitions_test.rs

```diff
@@ -0,0 +1,213 @@
+use fvm_ipld_blockstore::MemoryBlockstore;
+use fvm_ipld_encoding::ipld_block::IpldBlock;
+use fvm_shared::address::Address;
+use fvm_shared::bigint::Zero;
+use fvm_shared::econ::TokenAmount;
+use fvm_shared::error::ExitCode;
+use fvm_shared::sector::{RegisteredSealProof, SectorNumber};
+
+use fil_actor_cron::Method as CronMethod;
+use fil_actor_market::Method as MarketMethod;
+use fil_actor_miner::{
+    power_for_sector, DeadlineInfo, Method as MinerMethod, MovePartitionsParams,
+    ProveCommitSectorParams, State as MinerState,
+};
+
+use fil_actor_power::{Method as PowerMethod, State as PowerState};
+use fil_actors_integration_tests::expects::Expect;
+use fil_actors_integration_tests::util::{
+    advance_by_deadline_to_epoch, advance_to_proving_deadline, assert_invariants, create_accounts,
+    create_miner, cron_tick, get_network_stats, make_bitfield, miner_balance, precommit_sectors,
+    submit_windowed_post,
+};
+use fil_actors_runtime::runtime::Policy;
+use fil_actors_runtime::{
+    CRON_ACTOR_ADDR, CRON_ACTOR_ID, STORAGE_MARKET_ACTOR_ADDR, STORAGE_POWER_ACTOR_ADDR,
+    STORAGE_POWER_ACTOR_ID, SYSTEM_ACTOR_ADDR,
+};
+use test_vm::TestVM;
+use vm_api::trace::ExpectInvocation;
+use vm_api::util::{apply_ok, get_state, DynBlockstore};
+use vm_api::VM;
+
+#[test]
```

Comment:
In order to export tests to be run in other frameworks, test entry points no longer live here. Instead, they go in the test_vm package with a simple wrapper that invokes a method here with a single VM parameter. Please follow up to use the same pattern as other integration tests.